### ISSUE TITLE
Kover reporter failed

### ISSUE THREAD

**Issue by Tihon-Ustinov** [Kotlin/kotlinx-kover#46]
I have next default settings.

![image](https://user-images.githubusercontent.com/15249713/140751147-a93aa8c6-1296-4c67-b636-b97312be0a40.png)

When I run "koverReport" I get error.

![image](https://user-images.githubusercontent.com/15249713/140751330-f9e5c1d6-02e8-4c84-a16e-4c9e1e8a104d.png)

**Comment by shanshin** [comment]
Hi,
as I see task `jvmTest` is failed before `koverHtmlReport`.
Are these errors somehow related to Kover?

**Comment by shanshin** [comment]
I assume the errors in the tests arose due to the fact that you simultaneously applied the `kover` and `jacoco` plugins.

In most cases, tests become inoperable when they are instrumented by two agents at once.

**Comment by Tihon-Ustinov** [comment]
`jvmTest` is have only stdout errors (business logic), is's not breaking errors.

**Comment by shanshin** [comment]
> jvmTest is have only stdout errors (business logic), is's not breaking errors.

`13 errors` - are these expected errors in tests?

**Comment by Tihon-Ustinov** [comment]
> > jvmTest is have only stdout errors (business logic), is's not breaking errors.
> 
> `13 errors` - are these expected errors in tests?

Yes, I will try to remove jacoco tomorrow and report back on the results.

**Comment by Tihon-Ustinov** [comment]
> > jvmTest is have only stdout errors (business logic), is's not breaking errors.
> 
> `13 errors` - are these expected errors in tests?

I removed `jacoco` and disabled the error logs in the tests, but nothing helps.

I noticed that `kover` find file `test.ic.smap` in `build/kover/ompo-sdk/` but it is not there, there is `jvmTest.ic.smap`.
Could there be a problem with the filename?

**Comment by Tihon-Ustinov** [comment]
> > > jvmTest is have only stdout errors (business logic), is's not breaking errors.
> > 
> > 
> > `13 errors` - are these expected errors in tests?
> 
> I removed `jacoco` and disabled the error logs in the tests, but nothing helps.
> 
> I noticed that `kover` find file `test.ic.smap` in `build/kover/ompo-sdk/` but it is not there, there is `jvmTest.ic.smap`. Could there be a problem with the filename?

Yes, I changed the name to `jvm Test.is.snap` and it worked.

**Comment by shanshin** [comment]
This is an abnormal situation when the plugin works only after specifying the name of the report file.
The standard name for a test task in Kotlin/Multiplatform is `jvmTest`, but for some reason the report expects a file named `test`.
Can I see the project code or a small reproducer?

**Comment by Tihon-Ustinov** [comment]
> This is an abnormal situation when the plugin works only after specifying the name of the report file. The standard name for a test task in Kotlin/Multiplatform is `jvmTest`, but for some reason the report expects a file named `test`. Can I see the project code or a small reproducer?

I will try to reproduce this on a simple project and report the result

**Comment by shanshin** [comment]
@nomisRev, how did you  apply the plugin? 
Locally I got errors: 
```
> Could not resolve all artifacts for configuration ':allSourceSetsCompileDependenciesMetadata'.
   > Could not find org.jetbrains.kotlin:kotlin-stdlib-common:1.5.31.
     Searched in the following locations:
       - https://packages.jetbrains.team/maven/p/ij/intellij-dependencies/org/jetbrains/kotlin/kotlin-stdlib-common/1.5.31/kotlin-stdlib-common-1.5.31.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :
   > Could not find io.arrow-kt:arrow-core:1.0.1.
     Searched in the following locations:
       - https://packages.jetbrains.team/maven/p/ij/intellij-dependencies/io/arrow-kt/arrow-core/1.0.1/arrow-core-1.0.1.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :
   > Could not find io.arrow-kt:arrow-fx-coroutines:1.0.1.
     Searched in the following locations:
       - https://packages.jetbrains.team/maven/p/ij/intellij-dependencies/io/arrow-kt/arrow-fx-coroutines/1.0.1/arrow-fx-coroutines-1.0.1.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :

```

**Comment by nomisRev** [comment]
@shanshin I pushed the changes.
Here is the commit that adds Kover. https://github.com/nomisRev/Saga/commit/3e3cb5751a0f2bc8771103690547587035c3a130

**Comment by shanshin** [comment]
@nomisRev Thanks for the reproducer! 
The fix will be released soon.

**Comment by nomisRev** [comment]
Awesome! Glad I was able to help.

**Comment by shanshin** [comment]
Fixed in `0.4.2`
